Sources/Fhir/Resources/BodyMassContainer.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  BodyMassContainer.swift
+//  HealthKitOnFhir
+//
+//  Copyright (c) Microsoft Corporation.
+//  Licensed under the MIT License.
+
+import Foundation
+import HealthDataSync
+import HealthKit
+import HealthKitToFhir
+import FHIR
+
+open class BodyMassContainer : ResourceContainer<Observation>, HDSExternalObjectProtocol {
+    internal let unit = "lb"
```

Comment:
Is it always lb?  On my iPhone it doesn't look like I can change but was wondering if iOS is ever localized for kg when the market uses metric instead of imperial.

Reply:
iOS stores the values as an [HKUnit](https://developer.apple.com/documentation/healthkit/hkunit/1615733-unitfromstring?language=objc) which can be converted into several different mass types (grams, ounces, pounds, stones, etc.). We are just picking a type to use to send to FHIR - we could use something else if we want.

Reply:
I decided to normalize around metric values.